### ISSUE TITLE
Crash at initialization when compiling with go 1.23

### ISSUE THREAD

**Issue by rnestler** [ninech/nctl#158]
When compiling nctl with go 1.23 I get the following panic at startup:
```
panic: pattern "GET /debug/pprof/" (registered at net/http/pprof/pprof.go:100) conflicts with pattern "/debug/pprof/delta_heap" (registered at github.com/grafana/pyroscope-go/godeltaprof@v0.1.7/http/pprof/pprof.go:24):
	GET /debug/pprof/ matches fewer methods than /debug/pprof/delta_heap, but has a more general path pattern
```

https://github.com/grafana/agent/issues/6972 and https://github.com/grafana/alloy/issues/1540 seem to be the same error.

It seems to be fixed in https://github.com/grafana/pyroscope-go/releases/tag/godeltaprof%2Fv0.1.8

**Comment by rnestler** [comment]
I think it was already fixed in https://github.com/ninech/nctl/commit/0c54c5b96690bdc6c8171ae9e2c018fd547874c0 so only a release is necessary.

**Comment by thirdeyenick** [comment]
We just released [version 1.74](https://github.com/ninech/nctl/releases/tag/v1.7.4) which fixes this issue. Thanks for reporting!

**Comment by rnestler** [comment]
Thanks, seems to fix it :slightly_smiling_face:
